app/javascript/analytics/dashboard.js

```diff
@@ -44,7 +44,16 @@ function writeCards(data, timeRangeLabel) {
   followerCard.innerHTML = cardHTML(follows, `Followers ${timeRangeLabel}`);
 }
 
-function drawChart({ id, title, labels, datasets }) {
+function drawChart({ id, timeRangeLabel, title, labels, datasets }) {
+  // timeRangeLabel is '' when Infinity time range is selected
+  if (timeRangeLabel === '') {
+    // Remove dots from chart
+    datasets = datasets.map((dataset) => ({
```

Comment:
I'm unfamiliar with the code for rendering, but my read on this code is that we tell every dataset element to render with a `pointRadius` of 0.  Is that something we can set as a property of the chart? (So we don't need to loop on what we call an "infinity" of datasets)

Non-blocking, but something I'd like a front-end engineer to weigh in on.

Reply:
Well, actually a dataset is each line-chart. But I'm looking for a prettier way to do this!

Reply:
Thanks for the clarification, my assumption around the variable name `dataset` was that this was the array of points rendered on the chart.